src/pymor/models/iosys.py

```diff
@@ -800,13 +812,24 @@ class TransferFunction(InputOutputModel):
         The complex derivative of the transfer function.
     """
 
-    def __init__(self, input_space, output_space, H, dH, cont_time=True, cache_region='memory', name=None):
+    def __init__(self, input_space, output_space, tf, dtf, cont_time=True, cache_region='memory', name=None):
```

Comment:
This is an interface change, so a separate PR would make the change more visible.

Reply:
This is only to make the `__repr__` work, which uses `__signature__`. Actually, another/better option would be to make `H` and `dH` the attributes (instead of `tf` and `dtf`)...

Reply:
Still, it is an interface change which should not be buried in a PR with this title.